packages/bundle-size/src/reporters/cliReporter.test.js

```diff
@@ -0,0 +1,37 @@
+// Mock "chalk" to avoid special characters in output
+/** @param {string} val */
+function colorNoop(val) {
+  return val;
+}
+
+jest.mock('chalk', () => ({
+  bold: colorNoop,
+
+  cyan: colorNoop,
+  green: colorNoop,
+  red: colorNoop,
+}));
+
+const cliReporter = require('./cliReporter');
+const sampleReport = require('./sampleReport');
+
+describe('cliReporter', () => {
+  it('renders a report to CLI output', async () => {
+    const log = jest.spyOn(console, 'log').mockImplementation(() => {
```

Comment:
lest use implementation to express intent instead of comments

```suggestion
    const log = jest.spyOn(console, 'log').mockImplementation(noop)
```

Reply:
3dcca65